### PR TITLE
Assignment Complete

### PR DESCRIPTION
**Pull request by lsimmons1832**

### CODE REVIEW

**Comment by lsimmons1832** [comment]
@zoeames @morecallan @rousell

File: html/about.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	    	</article>
+	  	</section>
+	  </div>
+	  <div>
+		<img class="me" src="../CSS/images/Me.jpg"> 
+		</div>
+		<footer>
+			<a href="http://twitter.com/LuxVacabyTasha"><img src="../css/images/twitter.png" alt="Twitter Logo" class="social-icon"></a>
+        <a href="http://facebook.com/LuxuryVacationsbyTasha"><img src="../css/images/facebook.png" alt="Facebook Logo" class="social-icon"></a>
```
Comment (zoeames) [inline html/about.html]: remember this is you developer site so don't include personal pages.  you should have links to github, twitter, linkedin

File: html/contact.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <div>
+      <section class="contact">
+        <article>
+          <h2>Contact Details</h2>
+          <ul class="contact-info">
+            <li class="phone"><a href="tel: 615-777-9311">615-777-9311</a></li>
```
Comment (zoeames) [inline html/contact.html]: remove your phone number

File: html/contact.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <li class="mail"><a href="mailto:luxuryvacationsbytasha@gmail.com">luxuryvacationsbytasha@gmail.com</a></li>
```
Comment (zoeames) [inline html/contact.html]: make sure these are all dev specific.  also you seem to have them in your footer so maybe they aren't needed here?

File: html/contact.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      </section>
+    </div>
+    <div class="placeholder">
+      <section>
+        <article>
+          <p>Curabitur et accumsan diam. Nunc vitae venenatis lacus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Vestibulum pretium risus augue, sit amet placerat dui rhoncus in. Pellentesque eget quam vitae turpis consectetur molestie. Etiam lobortis in elit in pretium. Integer rhoncus tellus vitae tincidunt placerat.</p>
+        </article>
+      </section>
+    </div>
+    <footer>
```
Comment (zoeames) [inline html/contact.html:45]: this footer shows up kinda big relative to the page.  i would make it smaller and put everything (links, copyright) on one line

**Comment by zoeames** [comment]
looks like your images folder is inside your css folder?  they should both be on the same level in your project

**Comment by zoeames** [comment]
a good start here!  make sure this site is all about your developer work - not your travel stuff.  there are some inconsistencies in style between your home page and other pages.  make ALL your footers the same on each page.

**Comment by zoeames** [comment]
also it looks like there is some padding top and left on your home page but no padding right.  either pad all the way around or don't pad at all

**Comment by zoeames** [comment]
Also you did this in ONLY 1 commit?  not good practice.

**Comment by lsimmons1832** [comment]
@zoeames 
I think I got all of the corrections in.  Let me know

**Comment by zoeames** [comment]
looks great! 👍 i will merge for you

**Pull request merged**